parm/config/gfs/config.resources.GAEA

```diff
@@ -21,6 +18,57 @@ case ${step} in
     esac
     ;;
 
+  "eupd")
+    # update ntasks to 80 and threads_per_task to 20
+    case ${CASE} in
+      "C768")
+        export ntasks=80
+        export threads_per_task=20
+        ;;
+      *)
+        ;;
+    esac
+    export tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    ;;
+
+  "analcalc")
+    # decrease tasks_per_node 127 to 64
+    case ${CASE} in
+      "C768")
+        export tasks_per_node=64
+        ;;
+      *)
+        ;;
+    esac
+    ;;
+
+  "upp")
+    # decrease tasks_per_node 120 to 60
+    case ${CASE} in
+      "C768")
+        export tasks_per_node=60
+        ;;
+      *)
+        ;;
+    esac
+    ;;
+
+  "fcst")
+    # increase WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_{GDAS,GFS}
+    case ${CASE} in
+      "C768")
+        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS=20
+        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=25
+        (( WRTTASK_PER_GROUP_PER_THREAD_GDAS = WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS * 6 ))
+        (( WRTTASK_PER_GROUP_PER_THREAD_GFS = WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS * 6 ))
+        export WRTTASK_PER_GROUP_PER_THREAD_GDAS
+        export WRTTASK_PER_GROUP_PER_THREAD_GFS
```

Comment:
@DavidBurrows-NCO `config.resources.GAEA` is sourced at the end of `config.resources`, so these values will not be incorporated into the `ntasks` calculation automatically.  You may need to source `config.resources` again after setting these values and assign a boolean to prevent infinite recursion.  Something like

```suggestion
        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS=20
        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=25
        (( WRTTASK_PER_GROUP_PER_THREAD_GDAS = WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS * 6 ))
        (( WRTTASK_PER_GROUP_PER_THREAD_GFS = WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS * 6 ))
        export WRTTASK_PER_GROUP_PER_THREAD_GDAS
        export WRTTASK_PER_GROUP_PER_THREAD_GFS
        if [[ "${gaea_sourced_resources:-false}" == false ]]; then
            export gaea_sourced_resources=true
            source "${EXPDIR}/config.resources" "${step}"
        fi
```